### ISSUE TITLE
Add the ability to edit HTTP response headers

### ISSUE THREAD

**Issue by L8X** [AdguardTeam/AdGuardHome#4783]
The ability to add headers like HSTS, and things like content security policies are almost essential to servers these days, as web threats get less advanced in the skill needed to pull them off.

The last thing I want is for someone to use external javascript on my AH installation to bruteforce into my dashboard. 

Proposed addition:

Under "General Settings" or "Encryption Settings", add the area to add custom HTTP response headers which should then be served with all requests to adguard-home.example.com

**Comment by ainar-g** [comment]
This sounds like a task for a reverse proxy like Nginx, to be honest.  Especially if a general mechanism for all headers is desired.

For HSTS, there is already #2998.

**Comment by ainar-g** [comment]
After discussing this and a few related issues, @ameshkov and I have decided that, at least at the current stage, HTTP-header management is not a thing we'd like to add to AdGuard Home.  There are cases where the header might depends on dynamic settings, like HSTS, but generally it's not something most users need.  And for those who need it, tt's relatively easy to add any headers in the reverse-proxy configuration.  (And a reverse proxy is what people running AGH publicly probably should use anyway.)

For everyone wanting to e.g. modify the HSTS header (or any other header) themselves, we have added the instruction to the [*Encryption* page on our Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki/Encryption#other-hdrs).
